### PR TITLE
A new PR

### PR DESCRIPTION
**Pull request by NTaylorMullen**

### CODE REVIEW

**Comment by NTaylorMullen** [comment]
This comment is on the PR

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ public class ChatHub : Hub
     {
         Clients.All.receiveMessage(new Message(from, content));
     }
-}
+}
```
Comment (NTaylorMullen) [inline README.md:19]: This is on the file changed

Comment (davidfowl) [inline README.md:19]: This sucks!

**Comment by davidfowl** [comment]
Wow this is awesom

**Comment by NTaylorMullen** [comment]
Pick this up!

**Comment by NTaylorMullen** [comment]
1

**Comment by NTaylorMullen** [comment]
2

**Comment by davidfowl** [comment]
one

**Comment by davidfowl** [comment]
two

**Comment by davidfowl** [comment]
rfrer

**Comment by davidfowl** [comment]
wffwwfwefwefe

**Comment by halter73** [comment]
bugfix?

**Comment by davidfowl** [comment]
addwdEDIT

**Comment by davidfowl** [comment]
This is an event.

File: README.md (hunk 1 above)
Comment (NTaylorMullen) [inline README.md:19]: It wooorkkkks

**Pull request closed without merging**